2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews', 'root_dir': '/Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project', 'sweep_id': 'f5rn8zd6', 'run_id': 'ryuhxri2', 'sweep_param_path': '/Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/sweep-f5rn8zd6/config-ryuhxri2.yaml'}
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_init.py:_log_setup():371] Logging user logs to /Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/run-20220117_132011-ryuhxri2/logs/debug.log
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/run-20220117_132011-ryuhxri2/logs/debug-internal.log
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_init.py:init():404] calling init triggers
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
config: {}
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [wandb_init.py:init():460] starting backend
2022-01-17 13:20:11,998 INFO    Thread-2  :13406 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-17 13:20:12,012 INFO    Thread-2  :13406 [backend.py:ensure_launched():216] starting backend process...
2022-01-17 13:20:12,022 INFO    Thread-2  :13406 [backend.py:ensure_launched():221] started backend process with pid: 13414
2022-01-17 13:20:12,024 INFO    Thread-2  :13406 [wandb_init.py:init():469] backend started and connected
2022-01-17 13:20:12,027 INFO    Thread-2  :13406 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
2022-01-17 13:20:12,035 INFO    Thread-2  :13406 [wandb_init.py:init():533] updated telemetry
2022-01-17 13:20:12,068 INFO    Thread-2  :13406 [wandb_init.py:init():563] communicating current version
2022-01-17 13:20:14,107 INFO    Thread-2  :13406 [wandb_init.py:init():568] got version response 
2022-01-17 13:20:14,107 INFO    Thread-2  :13406 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-17 13:20:14,337 INFO    Thread-2  :13406 [wandb_init.py:init():606] starting run threads in backend
2022-01-17 13:20:15,906 INFO    Thread-2  :13406 [wandb_run.py:_console_start():1810] atexit reg
2022-01-17 13:20:15,907 INFO    Thread-2  :13406 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-17 13:20:15,907 INFO    Thread-2  :13406 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-17 13:20:15,907 ERROR   Thread-2  :13406 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-01-17 13:20:15,909 INFO    Thread-2  :13406 [wandb_init.py:init():633] run started, returning control to user process
2022-01-17 13:20:35,066 INFO    MainThread:13406 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-17 13:20:35,073 INFO    MainThread:13406 [wandb_run.py:_restore():1752] restore
